### ISSUE TITLE
await-async-utils false positives

### ISSUE THREAD

**Issue by ahuth** [testing-library/eslint-plugin-testing-library#118]
The await-async-utils rule is finding function calls that aren't actually from testing library. For example, if I define a function `wait` in my test code:

```js
function wait(delay) {
  return new Promise((resolve) => {
    setTimeout(resolve, delay);
  });
}
```

The lint rule will match it since it has the same name as the testing library's async util.

Would it make sense to first detect the import of testing-library, before applying the rule?

**Comment by Belco90** [comment]
You are right: that rule should check if that wait actually comes from a testing library import. This is checked in other rules, but it seems it's not happening for this one.
